src/components/ui/table/table.tsx

```diff
@@ -47,6 +48,8 @@ export function Table<T extends { id: number }>({
                     ? "whitespace-nowrap w-1/20"
                     : wrapColumns.includes(column as string)
                     ? "whitespace-normal text-center"
+                    : fixColWidth.includes(column as string)
```

Comment:
this will affect other screens with Name and Email Address col. can we use a flag here whether fix column width should be applied?

Reply:
done